src/rez/rezconfig.py

```diff
@@ -45,17 +45,39 @@
 # The package search path. Rez uses this to find packages. A package with the
 # same name and version in an earlier path takes precedence.
 packages_path = [
-    "~/packages",           # locally installed pkgs, not yet deployed
+    "~/packages/int",       # locally installed internal pkgs, not yet deployed
+    "~/packages/ext",       # locally installed external pkgs, not yet deployed
     "~/.rez/packages/int",  # internally developed pkgs, deployed
     "~/.rez/packages/ext",  # external (3rd party) pkgs, such as houdini, boost
 ]
 
 # The path that Rez will locally install packages to when rez-build is used
-local_packages_path = "~/packages"
+# The target_packages_path controls which path to use.
+# Note: Can be a string for backwards compatibility.
+local_packages_path = {
+    "internal": "~/packages/int",
+    "external": "~/packages/ext",
+}
```

Comment:
Just a head's up - I can't accept this change, because it changes an existing default config value, which is a backwards compatibility issue. I will set these back to previous defaults.

Reply:
It seems I never addressed this one. Must have slipped my sight.